contrib/flatpak-manifest.json

```
@@ -0,0 +1,61 @@
+{
+    "id": "us.gridcoin.Gridcoin-Research",
+    "branch": "main",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-23.08",
+    "sdk": "org.kde.Sdk",
+    "command": "gridcoinresearch",
+    "rename-desktop-file": "gridcoinresearch.desktop",
+    "rename-icon": "gridcoinresearch",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "cleanup": [
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+			"name": "libzip",
+			"cleanup": [ "/bin", "/share" ],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://nih.at/libzip/libzip-1.1.3.tar.xz",
+					"sha256": "729a295a59a9fd6e5b9fe9fd291d36ae391a9d2be0b0824510a214cfaa05ceee"
+				}
+			]
+		},
+        {
+            "name": "boost",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./bootstrap.sh --prefix=/app --with-libraries=system,filesystem,thread,iostreams,test",
+                "./b2 -j $FLATPAK_BUILDER_N_JOBS install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://boostorg.jfrog.io/artifactory/main/release/1.78.0/source/boost_1_78_0.tar.bz2",
+                    "sha256": "8681f175d4bdb26c52222665793eef08490d7758529330f98d3b29dd0735bccc"
+                }
+            ]
+        },
+        {
+            "name": "Gridcoin-Research",
+            "buildsystem": "autotools",
+            "config-opts": ["--with-boost-libdir=/app/lib"],
+            "sources": [
+                {
+                    "type": "dir",
+                    "path": ".."
+                }
+            ]
```

```suggestion
            "sources": [
                {
                    "type": "dir",
                    "path": ".."
                },
                {
                    "type": "shell",
                    "commands": ["./autogen.sh"]
                },
            ]
```
Getting: `bwrap: execvp ./configure: No such file or directory`